online-rest-api/src/main/java/eu/itesla_project/online/rest/api/impl/ProcessApiServiceImpl.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) 2016, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package eu.itesla_project.online.rest.api.impl;
+
+import java.text.SimpleDateFormat;
+import java.util.Objects;
+
+import eu.itesla_project.online.rest.model.Process;
+import eu.itesla_project.online.rest.model.WorkflowResult;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import javax.ws.rs.core.SecurityContext;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.joda.JodaModule;
+import eu.itesla_project.online.rest.api.DateTimeParameter;
+import eu.itesla_project.online.rest.api.ProcessApiService;
+import eu.itesla_project.online.rest.api.util.ProcessDBUtils;
+
+/**
+ *
+ * @author Giovanni Ferrari <giovanni.ferrari@techrain.it>
+ */
+public class ProcessApiServiceImpl extends ProcessApiService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ProcessApiServiceImpl.class);
+
+    private final ProcessDBUtils utils;
+    private final ObjectMapper objectMapper;
+    private final SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSXXX");
+
+    public ProcessApiServiceImpl(ProcessDBUtils utils) {
+        this.utils = Objects.requireNonNull(utils);
+        objectMapper = new ObjectMapper();
+        objectMapper.registerModule(new JodaModule());
+        objectMapper.configure(com.fasterxml.jackson.databind.SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false);
+        objectMapper.setDateFormat(sdf);
+    }
+
+    @Override
+    public Response getProcessList(String owner, String basecase, String name, DateTimeParameter date,
+            DateTimeParameter creationDate, SecurityContext securityContext) {
```

Comment:
The SecurityContext is injected by the container. 
The other parameters are nullable by design, it should be possible to filer the process list only on a subset of the parameters.